code/game/objects/effects/landmarks.dm

```diff
@@ -48,6 +48,9 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 			GLOB.jobspawn_overrides[name] = list()
 		GLOB.jobspawn_overrides[name] += src
 	..()
+
+/obj/effect/landmark/start/Initialize()
+	. = ..()

```

Comment:
Not sure about this, if the other stuff is needed right away than the tag might be as well